app/ydoc-shared/src/ast/tree.ts

```diff
@@ -3634,10 +3654,12 @@ function setNode<
   map.set(key, updated as Fields[Key])
 }
 
-function autospaced<T extends object | string>(node: T): NodeChild<T>
-function autospaced<T extends object | string>(node: T | undefined): NodeChild<T> | undefined
+export function autospaced<T extends object | string>(node: T): NodeChild<T>
+export function autospaced<T extends object | string>(node: T | undefined): NodeChild<T> | undefined
 /** TODO: Add docs */
-function autospaced<T extends object | string>(node: T | undefined): NodeChild<T> | undefined {
+export function autospaced<T extends object | string>(
```

Comment:
Why export this? I prefer to keep all whitespace-creation logic in the AST APIs, not the responsibility of the other application logic